nixos/modules/services/admin/nomachine.nix

```diff
@@ -0,0 +1,258 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.nxserver;
+  nxserver = pkgs.nomachine;
+
+  preStartScript = pkgs.writeScript "nxserver-prestart" ''
+    #!${pkgs.runtimeShell}
+
+    if [ ! -f /var/lib/nxserver/etc/version ]; then
+      cp ${nxserver}/NX/etc.static/version /var/lib/nxserver/etc/
+    fi
+    if [ ! -f /var/lib/nxserver/etc/usb.db ]; then
+      cp ${nxserver}/NX/etc.static/usb.db /var/lib/nxserver/etc/
+    fi
+    if [ ! -f /var/lib/nxserver/etc/update.cfg ]; then
+      cp ${nxserver}/NX/etc.static/update.cfg /var/lib/nxserver/etc/
+    fi
```

Comment:
These seem like files we don't want to keep between runs, and should preferably be stored in the RuntimeDirectory, or at least overwritten on start.